lib/PhpStatistics.php

```diff
@@ -64,4 +66,67 @@ protected function cleanVersion($version) {
 		}
 		return $version;
 	}
+
+	/**
+	 * Get status information about the cache from the OPcache extension
+	 *
+	 * @return array with an array of state information about the cache instance
+	 */
+	protected function getOPcacheStatus() {
```

Comment:
I would prefer to return a empty array if no information is found. That makes it possible add the return type.

Reply:
I was inspired by SystemStatistics.php. It uses 'N/A' for all undefined values. I prefer returning a defined value for monitoring systems. It makes it easier to distinguish between a value not being available ('N/A') or some other error (empty values).

Reply:
I don't see how mixing types (string & array) makes it easier to distinguish.